demo/BERT/scripts/inference_benchmark_megatron.sh

```diff
@@ -27,7 +27,7 @@ WEIGHTS="${7}"
 
 CHECKPOINTS_DIR="models/fine-tuned/bert_tf_ckpt_${MODEL_VARIANT}_qa_squad2_amp_${SEQUENCE_LENGTH}_v19.03.1"
 SQUAD_DIR="BERT/squad"
-ENGINE_NAME="engines/bert_mt_${MODEL_VARIANT}_${PRECISION}_bs${MAX_BATCH}_seqlen${SEQUENCE_LENGTH}_benchmark.engine"
+ENGINE_NAME="engines/bert_mt_${MODEL_VARIANT}_${PRECISION}_bs${MAX_BATCH}_seqlen${SEQUENCE_LENGTH}_weight${WEIGHTS}_benchmark.engine"
```

Comment:
@ttyio is this correct?

Reply:
Yes, it's to differentiate between dense and sparse weight engines